src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SwaggerRoutes.scala

```diff
@@ -67,4 +70,20 @@ trait SwaggerRoutes {
     }
   }
 
+  // Make Swagger UI respect the x-tokenName security scheme extension.
+  // We have to string replace it's unfortunately hard-coded in Swagger UI. See:
+  // https://github.com/swagger-api/swagger-ui/blob/cc408812fc927e265da158bf68239530740ab4cc/src/core/oauth2-authorize.js#L25
+  private val serveSwaggerUiBundle: server.Route = {
+    mapResponseEntity { entityFromJar =>
+      entityFromJar.transformDataBytes(Flow.fromFunction { original =>
+        ByteString(
+          original.utf8String
+            .replace(""""response_type=token"""",
+              """(t.name === "b2coauth" ? "response_type=id_token&nonce=defaultNonce&prompt=login" : "response_type=token")"""))
```

Comment:
Had to do this slightly differently than [TDR](https://github.com/DataBiosphere/jade-data-repo/blob/mod_oauth2_poc/src/main/java/bio/terra/app/configuration/WebConfig.java#L36-L41). The UI must be slightly different, I couldn't find a `x-tokenName` property in the JS object. We can probably remove the `t.name === "b2coauth"` conditional when we're ready to only use B2C.